test/build.sh

```diff
@@ -138,7 +138,7 @@ function build-ci() {
   export REGISTRY="${NPD_STAGING_REGISTRY}/ci"
   export VERSION="$(get-version)-$(date +%Y%m%d.%H%M)"
   export TAG="${VERSION}"
-  make push
+  make push-tar
```

Comment:
/hold

instead of reverting the test-infra change, let's switch to only `push-tar` here but add building but not pushing

Reply:
SGTM